app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>List of Users</h1>
+<h1>Listing users</h1>
+<div class="container">
+  <h2>Users</h2>
+  <% @users.each do |user| %>
+    <%= link_to user_path(user) do %>
```

Comment:
- [ ] When a user is clicked, the URL slug is supposed to be `https://127.0.0.1/users/user_id/posts` and not `https://users/user_id`, Kindly make sure you fix that.
- [ ] Also the design is supposed to match the wireframe provided which you can find below :point_down: 
![image](https://github.com/saluumaa/blog-app/assets/97100091/4e08f208-b4e3-4847-b1bd-cf30cea810d8)

You can ignore the pagination for now :smile: